### PR TITLE
Prepare for 11.0.9 EA builds cycle

### PR DESCRIPTION
**Pull request by jhuttana**
Could you please review the changes?

### CODE REVIEW

File: install-rhel-deps-build-openjdk11.sh [hunk 1]
```diff
@@ -62,11 +62,11 @@ EOF
...
 pushd /opt
-wget -O jdk-11.0.7_10.tar.gz "https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-11.0.7%2B10/OpenJDK11U-jdk_$(platform_name)_11.0.7_10.tar.gz"
-tar -xf jdk-11.0.7_10.tar.gz
-/opt/openjdk-11.0.7+10/bin/java -version
+wget -O jdk-11.0.8_10.tar.gz "https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-11.0.8%2B10/OpenJDK11U-jdk_$(platform_name)_11.0.8_10.tar.gz"
+tar -xf jdk-11.0.8_10.tar.gz
+/opt/openjdk-11.0.8+10/bin/java -version
 popd
-BOOT_JDK="/opt/openjdk-11.0.7+10/"
+BOOT_JDK="/opt/openjdk-11.0.8+10/"
```
Comment (jerboaa) [inline install-rhel-deps-build-openjdk11.sh]: Have you tried this? AFAIK, folder names now are `openjdk-11.0.8_10` NOT `openjdk-11.0.8+10`.

**Review by jerboaa — CHANGES_REQUESTED** [review]

File: install-rhel-deps-build-openjdk11.sh (hunk 1 above)
Comment (jhuttana) [reply install-rhel-deps-build-openjdk11.sh]: You are right :) It was my mistake ;) Now I have addressed the review comment.
Could you please take a look?

**Review by jerboaa — COMMENTED** [review]
LGTM. Like JDK 8, we should only merge once 11.0.9+1 has been tagged upstream.

**Comment by jhuttana** [comment]
> LGTM. Like JDK 8, we should only merge once 11.0.9+1 has been tagged upstream.

Sure. Thank you!

**Review by jerboaa — APPROVED** [review]
LGTM

**Pull request merged**